src/atoms/connectWalletButton/connectWalletButton.styles.ts

```diff
@@ -0,0 +1,34 @@
+import { Styled } from '../../theme';
```

Comment:
@shan8851  i think its a mistake that connect btn using different font-family, can we put same like everywhere